suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -273,11 +275,7 @@ export const getAccountTypeUrl = (path: string) => {
     }
 };
 
-export const getAccountDecimals = (symbol: NetworkSymbol) => {
-    const network = networksCompatibility.find(n => n.symbol === symbol);
-
-    return network?.decimals;
-};
+export const getAccountDecimals = (symbol: NetworkSymbol) => networks[symbol]?.decimals;
```

Comment:
the `?.` is not necessary TS-wise, but there'd be runtime errors, because this functions gets called with with string cast as `NetworkSymbol` :disappointed:

Reply:
This would still create some unexpected behaviour, then. It might be better to just let the app crash in those cases so that QA or Sentry can catch it.

Reply:
As discussed in PM, we shall postpone it (usages of `getAccountDecimals` rely on this behavior)